src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-redis.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: Collector for Redis
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - Redis
+metaDescription: You can collect redis metrics using the OpenTelemetry collector.
+---
+import infrastructureRedisOTELDashboard from 'images/infrastructure_screenshot-crop_redis-otel-dashboard.webp'
+
+You can collect metrics about your Redis instance with the OpenTelemetry collector. The collector is a component of OpenTelemetry that collects, processes, and exports telemetry data to New Relic (or any observability backend).
+
+The Redis receiver is designed to retrieve Redis INFO data from a single Redis instance, build metrics from that data, and send them to the next consumer at a configurable interval.
+
+You can view this [metric data](#data) in pre-built dashboards, create alert policies, and create custom queries and charts.
+
+<img
+  title="Redis OpenTelemetry quickstart dashboard"
+  alt="Image of the dashboard available through the Redis OpenTelemetry quickstart"
+  src={infrastructureRedisOTELDashboard}
+/>
+
+
+<Callout variant="tip">
+  If you're looking for help with other collector use cases, see the [newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples) repository.
+</Callout>
+
+Complete the steps below to collect metrics from Redis:
+
+## Step 1: Sign up for your free account if you haven't already [#signup]
+
+<InlineSignup />
+
+## Step 2: Prerequisites [#prerequisites]
+
+The collector will use the [redis receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/redisreceiver) component (minimum version recommended v0.83.0), and it requires access to Redis through an endpoint.
+
+## Step 3: Install OpenTelemetry collector [#install-opentelemetry-collector]
+
+Download and install the OpenTelemetry collector following [OpenTelemetry docs](https://opentelemetry.io/docs/collector/getting-started/).
+
+You will need to install [OpenTelemetry Collector Contrib Distro](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib) or other distribution including, at least, the following components:
+
+* [Redis receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/redisreceiver)
```

Comment:
Each one of those needs to be present in the otel distribution otherwise they would not be able to apply the suggested configuration. 
However, they do not need to install each component, an otel distro either has them or not.

For example the newrelic distro (nrdot) has a subset of all the components available in the "contrib" distro